drag_and_drop_v2/default_data.py

```diff
@@ -83,19 +88,34 @@
         "incorrect": ITEM_INCORRECT_FEEDBACK,
         "correct": ITEM_CORRECT_FEEDBACK.format(zone=BOTTOM_ZONE_TITLE)
       },
-      "zone": BOTTOM_ZONE_ID,
+      "zones": [
+        BOTTOM_ZONE_ID
+      ],
       "imageURL": "",
       "id": 2,
     },
     {
+      "displayName": _("Goes anywhere"),
+      "feedback": {
+        "correct": ITEM_ANY_ZONE_FEEDBACK
```

Comment:
@haikuginger This is missing an `"incorrect": "",` entry; please add that above this line.